pkg/kv/kvserver/stateloader/stateloader.go

```diff
@@ -374,12 +374,13 @@ func (rsl StateLoader) SetForceFlushIndex(
 // empty storage.
 func UninitializedReplicaState(rangeID roachpb.RangeID) kvserverpb.ReplicaState {
 	return kvserverpb.ReplicaState{
-		Desc:           &roachpb.RangeDescriptor{RangeID: rangeID},
-		Lease:          &roachpb.Lease{},
-		TruncatedState: nil, // Invariant: always nil.
-		GCThreshold:    &hlc.Timestamp{},
-		Stats:          &enginepb.MVCCStats{},
-		GCHint:         &roachpb.GCHint{},
+		LeaseAppliedIndex: InitialLeaseAppliedIndex,
```

Comment:
Should we leave this at zero like other fields? I'm not sure though that some proposals at low LAIs wouldn't slip through. Maybe they wouldn't since this range is not initialized yet.

Reply:
Reversed to 0 - for uninited replicas, this index has no semantics. And the `max(..., InitialLeaseAppliedIndex)` in proposal buf `Init` is the best guarantee we won't propose low LAIs.